website/documentation/guides/administer-shoots/scalability/_index.md

```diff
@@ -0,0 +1,141 @@
+- [Kubernetes Scalability Thresholds](https://github.com/kubernetes/community/blob/master/sig-scalability/configs-and-limits/thresholds.md)
+
+Furthermore, the problem space has been discussed in a [kubcon talk](https://www.youtube.com/watch?v=t_Ww6ELKl4Q), which slides can be found [here](https://docs.google.com/presentation/d/1aWjxpY4YJ4KJQUTqaVHdR4sbhwqDiW30EF4_hGCc-gI). You should at least read the slides before continuing.
+
+Essentially, it comes down to this:
+
+>If you promise to:
+> - correctly configure your cluster
+> - use extensibility features "reasonably"
+> - keep the load in the cluster within recommended limits
+>
+>Then we promise that your cluster will function properly.
+
+With that knowledge in mind, let's look at Gardener and eventually pick up the question about the number of objects in ETCD raised above.
+
+## Gardener-specific considerations
+
+The following considerations are based on experience with various large clusters that scaled in different dimensions. Just as explained above, pushing beyond even one of the limits is likely to cause issues at some point in time (but not guaranteed). Depending on the setup of your workloads however, it might work unexpectedly well. Nevertheless, we urge you take conscious decisions and rather think about sharding your workloads. Please keep in mind - your workload affects the overall stability and scalability of a cluster significantly.
+
+### ETCD
+
+**The following section is based on a setup where ETCD `Pods` run on a dedicated `Node` pool and each `Node` has 8 vCPU and 32GB memory at least.**
+
+ETCD has a practical space limit of 8 GB. It caps the number of objects one can technically have in a Kubernetes cluster.
+
+Of course, the number is heavily influenced by each object's size, especially when considering that secrets and configmaps may store up to 1MB of data. Another dimension is a cluster's churn rate. Since ETCD stores a history of the keyspace, a higher churn rate reduces the number of objects. Gardener runs [compaction](https://etcd.io/docs/v3.4/op-guide/maintenance/#history-compaction) every 30min and [defragmentation](https://etcd.io/docs/v3.4/op-guide/maintenance/#defragmentation) once per day during a cluster's maintenance window to ensure proper ETCD operations. However, it is still possible to overload ETCD. If the space limit is reached, ETCD will only accept `READ` or `DELETE` requests and manual interaction by a Gardener operator is needed to disarm the alarm, once you got below the threshold.
+
+To avoid such situation, you can monitor the current ETCD usage via the "ETCD" dashboard of the monitoring stack. It gives you the current DB size as well as historical data for the past 2 weeks. While there are improvements planned to trigger compaction and defragmentation based on DB size, an ETCD should not grow up to this threshold. A typical, healthy DB size is less than 3 GB.
```

Comment:
```suggestion
To avoid such a situation, you can monitor the current ETCD usage via the "ETCD" dashboard of the monitoring stack. It gives you the current DB size, as well as historical data for the past 2 weeks. While there are improvements planned to trigger compaction and defragmentation based on DB size, an ETCD should not grow up to this threshold. A typical, healthy DB size is less than 3 GB.
```